README.md

```diff
@@ -138,6 +138,13 @@ Please use the template below when updating the changelog:
 * r/tfe_resource: description of change or bug fix ([#124](link-to-PR))
 ```
 
+### Updating the documentation.
+
+To update the documentation for the provider, please copy the new markdown and
+paste it here: https://registry.terraform.io/tools/doc-preview
+
+Then copy/paste an image of the updated documentation in your pull request.
```

Comment:
This reads like it's instructions for updating the docs, but it's not, quite; it's just an extra thing we'd like people to do if their PR involves docs. How about this?

```suggestion
### Updating the documentation

For pull requests that update provider documentation, please help us verify that the
markdown will display correctly on the Registry:

- Copy the new markdown and paste it here to preview: https://registry.terraform.io/tools/doc-preview
- Paste a screenshot of that preview in your pull request.
```